tests/integration_tests/functional/test_cpu_template_helper.py

```diff
@@ -277,6 +281,8 @@ def test_cpu_config_dump_vs_actual(
             f"{actual=:#034b} vs. {dump=:#034b}"
         )
 
+    assert len(keys_not_in_dump) == 0
```

Comment:
Shouldn't this be `assert len(keys_not_in_dump) == 0, keys_not_in_dump` or `assert not keys_not_in_dump` to actually get a print of them? Or do we get the whole list anyway due to pytest debug output in case of test failure?

Reply:
ohhh, neat! Has it always done that? :o don't think I've seen this before :o